Bhavika/html/Assignment6/Ass6.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>CSS</title>
+        <link rel="stylesheet" type="text/css" href="style.css" />
+        <style type="text/css"></style>
+        <style>
+            body {
+                font-family: Arial, Helvetica, sans-serif;
+                background-color: #e2dfdc;
+            }
+            h1 {
+                border: 2px solid #91b1c5;
+                background-color: #91b1c5;
+            }
+            h1 {
+                text-align: right;
+                padding: 10px;
+            }
+            h2 {
+                background-color: #91b1c5;
+            }
+            h2 {
+                text-align: right;
+                padding: 10px;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>MI-9 Intern HTML Assesment</h1>
+        <h2><i>Personal Details</i></h2>
+        <label for="fname">First name:</label>
+        <input type="text" id="fname" name="fname" /><br />
+        <br />
+        <label for="lname">Last name:</label>
+        <input type="text" id="lname" name="lname" /><br />
+        <br />
+        <label for="Gender">Gender:</label>
+        <input type="radio" id="male" name="Gender" value="male" />
```

Comment:
check this really working or not may be its selected male as well as other options too